.github/workflows/tests.yaml

```diff
@@ -9,8 +9,8 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  unit-test:
-    name: Run Unit and Lint Tests
+  test:
```

Comment:
I believe renaming these tests would require updating required status checks, should we discuss the tests design and perhaps finalise them?